src/js/package.json

```diff
@@ -84,25 +80,13 @@
     "build-inner": "node esbuild.config.inner.mjs",
     "build": "tsc --noEmit && node esbuild.config.outer.mjs",
     "test": "npm-run-all test:*",
-    "test:unit": "cross-env TEST_NODE=1 ts-mocha --node-option=experimental-loader=./test/loader.mjs --node-option=experimental-wasm-stack-switching -p tsconfig.test.json \"test/unit/**\"",
-    "test:node": "cross-env TEST_NODE=1 mocha test/integration/**/*.test.js",
-    "test:browser": "mocha test/integration/**/*.test.js",
+    "test:unit": "cross-env TEST_NODE=1 TSX_TSCONFIG_PATH=tsconfig.test.json node --import tsx --import test/loader.mjs --experimental-wasm-stack-switching --test test/unit/**/*.test.*",
```

Comment:
I was wondering if some options are still valid now.

Passing the test doesn't depend on...
1. whether `tsconfig.test.json` is specified.
2. whether `test/loader.mjs` is specified.
3. where `--experimental-wasm-stack-switching` is specified.

If they aren't needed now, I can remove them.

Reply:
It could be changed to `--experimental-wasm-jspi` but indeed it's still needed until we node 26 comes out and we switch to that.